dashboard/config/routes.rb

```diff
@@ -571,6 +571,8 @@ module OPS
 
       post 'users/:user_id/post_ui_tip_dismissed', to: 'users#post_ui_tip_dismissed'
 
+      post 'users/:user_id/accept_data_transfer_agreement', to: 'users#accept_data_transfer_agreement'
```

Comment:
How does this relate to the `api/v1` route we're using elsewhere?

Reply:
This _is_ the `api/v1` route - expand above the diff to see the relevant namespace info.